web_auto_refresh/README.rst

```diff
@@ -0,0 +1,78 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
can you follow new README policy? https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst